concentration/Scene/SpinningBoxAndBalls.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  SpinningBoxAndBalls.swift
+//  concentration
+//
+//  Created by Olha Dzhyhirei on 1/30/23.
+//
+
+import SpriteKit
+
+class SpinningBoxAndBalls: SKScene {
+    
+    var BallQuantity = 16
+    
+    enum GameState { case startGame, playing }
+    var gameState = GameState.startGame
+    var box: SKShapeNode!
+    
+    override func didMove(to view: SKView) {
+        self.anchorPoint = .init(x: 0, y: 0)
+        self.scene?.backgroundColor = .link
+        setupStartLabel()
```

Comment:
It will be better if balls are set from the very begging so by first tap you just apply gravity ti them and start rotating the box

Reply:
done